src/compiler/AST/Typed.elm

```diff
@@ -32,7 +32,7 @@ type alias ProjectFields =
     { modules : Modules LocatedExpr }
 
 
-{-| Differs from Canonical.LocatedExpr by:
+{-| Differs from Canonical.Expr by:
```

Comment:
still at the wrong place :upside_down_face: 

instead of
```elm
{-| ... -}
type alias LocatedExpr

type alias Expr
```
I wanted
```elm
type alias LocatedExpr

{-| ... -}
type alias Expr
```
Sorry, the language barrier is probably showing :) :czech_republic: :fr: